templates/aerospike-cluster-entrypoint-new-vpc.template.yaml

```diff
@@ -76,12 +70,6 @@ Metadata:
         default: "Namespace file"
       EBS:
         default: "EBS volume size"
-      BastionAMIOS:
-        default: Bastion AMI operating system
-      BastionInstanceType:
-        default: Bastion instance type
-      NumBastionHosts:
-        default: Number of bastion hosts
```

Comment:
Should we not still let the customer choose how they want to configure these?

Reply:
done